sw/device/tests/dif/dif_rv_timer_sanitytest.c

```diff
@@ -28,7 +28,7 @@ static const uint32_t kHart = (uint32_t)kTopEarlgreyPlicTargetIbex0;
 static const uint32_t kComparator = 0;
 
 static const uint64_t kTickFreqHz = 1000 * 1000;  // 1 MHz.
-static const uint64_t kDeadline = 0x10000;        // 10 ms.
+static const uint64_t kDeadline = 0x20000;        // 20 ms.
```

Comment:
This is more like 128ms. Should be 20000 instead of 10000. 
Why 20ms though? This increases sim runtime.

Reply:
i should have clarified in the test. With the new clock frequency changes , the interrupt now triggers before the test gets to the "WFI" command.  And this basically causes the test to hang forever.  I should probably change how the test is done instead of just extending the time.

Reply:
I think hoisting the `LOG` out of the loop should fix any timing issues.

Reply:
thanks for reminding me, i meant to fix this differently and completely forgot.